media-sound/gmorgan/files/gmorgan-0.79-remove-old-docs.patch

```diff
@@ -0,0 +1,29 @@
+From 8e5908a5fde07776acbe186301009bab9d826884 Mon Sep 17 00:00:00 2001
+From: Harri Nieminen <moikkis@gmail.com>
+Date: Sat, 4 Mar 2017 14:18:13 +0200
+Subject: [PATCH] Remove old docs
+
+---
+ doc/Makefile.am | 9 +--------
+ 1 file changed, 1 insertion(+), 8 deletions(-)
+
+diff --git a/doc/Makefile.am b/doc/Makefile.am
+index 16187f0..3b16085 100644
+--- a/doc/Makefile.am
++++ b/doc/Makefile.am
+@@ -1,11 +1,4 @@
+ datadir = $(prefix)/share/${PACKAGE}/doc
+-data_DATA=gmorgan0.75.pdf \
+-gmorgan0.75.odt \
+-gmorgan-old-es.pdf \
+-gmorgan-old-es.sxw \
+-gmorgan-old-fr.pdf \
+-gmorgan-old-fr.sxw \
+-gmorgan-old.pdf \
+-gmorgan-old.sxw 
++data_DATA=gmorgan0.75.pdf
```

Comment:
also, you sure this shouldn't be `dist_doc_DATA`? PDF files should usually be installed into `$(docdir)`